lib/core/include/irods/library_features.h

```diff
@@ -21,42 +21,84 @@
 
 /// Defined if the development library supports the C++ ticket administration library.
 ///
+/// \sa Prefer IRODS_LIBRARY_FEATURE_LIB_TICKET_ADMINISTRATION
+///
 /// \since 4.3.1
-#define IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION                              202209L
+#define IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION                               202209L
+
+/// An alias for IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION.
+///
+/// \since 4.3.2
+#define IRODS_LIBRARY_FEATURE_LIB_TICKET_ADMINISTRATION                       202209L
```

Comment:
Would it make sense to have the alias macros defined by the other macro? In other words...
```c
#define IRODS_LIBRARY_FEATURE_LIB_TICKET_ADMINISTRATION                       202209L
#define IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION                               IRODS_LIBRARY_FEATURE_LIB_TICKET_ADMINISTRATION
```
- OR -
```c
#define IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION                               202209L
#define IRODS_LIBRARY_FEATURE_LIB_TICKET_ADMINISTRATION                       IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION
```

Reply:
I considered that and decided to keep them separate in case they end up diverging (on purpose).

Reply:
I'm going to make them aliases.

It just makes more sense from a maintainability perspective.

Reply:
Done.